### PR TITLE
Create dgps_noinitials.csl

### PR DESCRIPTION
**Pull request by jschoellbauer**
initial version
DGPs style but with authors' full first names instead of initials

### CODE REVIEW

**Comment by bwiernik** [comment]
Thanks! Is this format called for by a specific journal or style guide?

**Comment by jschoellbauer** [comment]
Yes, this style is requested to be used by the students studying Business Administration and Psychology at the Ferdinand Porsche FERNFH - Distance-Learning University of Applied Sciences (https://www.fernfh.ac.at/en/distance-learning/degree-programmes/ma-in-business-administration-and-psychology).

**Comment by bwiernik** [comment]
Can you please:

1. Re-title the style "Ferdinand Porsche FERNFH - Master Betriebswirtschaft und Wirtschaftspsychologie - DGPS"
2. Rename the file "ferdinand-porsche-fernfh-master-betriebswirtschaft-und-wirtschaftspsychologie-dgps.csl"
3. Update the documentation link to point to the style requirements on the university website?

Thank you

**Comment by jschoellbauer** [comment]
I made the recommended changes and now ask you to publish the style : )

**Comment by jschoellbauer** [comment]
Hello, can you please publish the style before the new study year begins? 🙏 Or do you need anything else from me? Best, Julia

File: ferdinand-porsche-fernfh-betriebswirtschaft-und-wirtschaftspsychologie-dgps.csl [hunk 1]
```diff
@@ -0,0 +1,373 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Ferdinand Porsche FERNFH - Betriebswirtschaft und Wirtschaftspsychologie - DGPS</title>
```
Comment (bwiernik) [inline ferdinand-porsche-fernfh-betriebswirtschaft-und-wirtschaftspsychologie-dgps.csl:5]:
```suggestion
    <title>Ferdinand Porsche FERNFH - Betriebswirtschaft und Wirtschaftspsychologie - DGPS (Deutsch)</title>
```

**Comment by bwiernik** [comment]
Thanks!

**Pull request merged**